### PR TITLE
Update README to  reflect switch to standalone

### PR DESCRIPTION
**Pull request by loiswells97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-Variables for the web component can be placed in `.env`.
-
 ## Private repo setup (.npmrc)
 
-The app requires a Git token for access to private repos (currently limited to `design-system-react`).
+The app requires a Git token for access to private repos (currently limited to [`design-system-react`](https://github.com/RaspberryPiFoundation/design-system-react).
 
-- Generated a token here, it'll be prefixed with `ghp_`: https://github.com/settings/tokens
+- Generate a token [here](https://github.com/settings/tokens) - it should be prefixed with `ghp_`
 - Add a line to the bottom of `~/.npmrc` (this is in addition to the one in the repo): `//npm.pkg.github.com/:_authToken=<github_token>`
```
Comment (sra405) [inline README.md:18]: 🔧 I don't think this is true anymore. Both design system packages are public I believe:
- https://www.npmjs.com/package/@raspberrypifoundation/design-system-react
- https://www.npmjs.com/package/@raspberrypifoundation/design-system-core

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
+### CHanged
```
Comment (sra405) [inline CHANGELOG.md:9]:
```suggestion
### Changed
```

File: README.md [hunk 3]
```diff
@@ -78,9 +71,11 @@ The web component can be included in a page by using the `<editor-wc>` HTML elem
 - `embedded`: Enable embedded mode which hides some functionality (defaults to `false`)
 - `output_split_view`: Start with split view in output panel (defaults to `false`, i.e. tabbed view)
```
Comment (sra405) [inline README.md:72]: 🌱 I _think_ there's more attributes than this now 👀

File: README.md [hunk 4]
```diff
@@ -78,9 +71,11 @@ The web component can be included in a page by using the `<editor-wc>` HTML elem
...
 
-It is possible to add query strings to control how the web component is configured. Any HTML attribute can be set on the query string, including `class`, `style` etc.
+## Development
+
+### Previewing
 
-For example, to load the page with the Sense Hat always showing, add [`?sense_hat_always_enabled` to the URL](http://localhost:3011?sense_hat_always_enabled)
+The web component test page at `http://localhost:3011` can be used to develop the web component in isolation if needed. This page is configured to pass query parameters into the web component as attributes (including `class`, `style` etc.), allowing the web component to be previewed in different states during development. For example, to preview the web component with the Sense HAT always showing, visit `http://localhost:3011?sense_hat_always_enabled=true`.
```
Comment (sra405) [inline README.md:78]:
```suggestion
The web component test page at `http://localhost:3011` can be used to develop the web component in isolation if needed. This page is configured to pass query parameters into the web component as attributes (including `class`, `style` etc.), allowing the web component to be previewed in different states during development. For example, to preview the web component with the Sense HAT always showing, visit `http://localhost:3011/web-component.html?sense_hat_always_enabled=true`.
```

File: README.md [hunk 5]
```diff
@@ -177,18 +172,23 @@ Python code snippets are styled and syntax-highlighted using the `language-pytho
...
 
 - `AWS_ACCESS_KEY_ID`
 - `AWS_REGION`
 - `AWS_S3_BUCKET`
 - `AWS_SECRET_ACCESS_KEY`
 
-Other variables that pertain to the app, rather than its deployment are set with defaults in the [build-and-deploy workflow](./.github/workflows/build-and-deploy.yml). These are also in `.env.example`.
+Other variables that pertain to the app, rather than its deployment, are set to default values in the [build-and-deploy workflow file](./.github/workflows/build-and-deploy.yml). These are also in `.env.example`.
+
+The staging bucket is called [`editor-dist-staging`](https://dash.cloudflare.com/44a2049cd9f2b11d21474e06251367df/r2/default/buckets/editor-dist-staging), and the latest deployment of `main` is available at https://staging-editor-static.raspberrypi.org/branches/main/web-component.html.
```
Comment (sra405) [inline README.md:184]: ❓ is it worth mentioning the staging js bundle is located at `https://staging-editor-static.raspberrypi.org/branches/main/web-component.js`?

File: README.md (hunk 1 above)
Comment (loiswells97) [reply README.md:18]: Ah yes okay, I might just get rid of this lot then 👍

File: README.md (hunk 3 above)
Comment (loiswells97) [reply README.md:72]: Ohh yes, I was planning to come back to fill this in and I forgot, thanks!

File: README.md (hunk 4 above)
Comment (loiswells97) [reply README.md:78]: Ah yes, thanks 😅 👍

File: README.md (hunk 5 above)
Comment (loiswells97) [reply README.md:184]: Ah okay, yeah, I might rework this bit a little then to make clear the bundle and preview locations 👍

**Review by sra405 — APPROVED** [review]
🙌  great changes!!

**Review by conorriches — APPROVED** [review]

**Review by KristinaDudnyk — APPROVED** [review]
🚀🚀🚀

**Pull request merged**